core/purchasing/inquiry/supplier_allocation_inquiry.php

```diff
@@ -20,20 +20,10 @@
 	$js .= get_js_open_window(900, 500);
 if (user_use_date_picker())
 	$js .= get_js_date_picker();
+$js .= get_js_history(array('supplier_id', 'TransAfterDate', 'TransToDate'));
```

Comment:
Note: The `FromDate` and `ToDate` form fields will now get renamed to `TransAfterDate` and `TransToDate` in line with useage in the codebase and will need to be taken care in the extensions and the RestAPIs as well in all files where such a change occurs.

Reply:
Yes, it appears like FromDate and ToDate were never set or used, so renaming them should be fine.

Note that the fields added to set_posts() actually improve the ability of FA to be called from another application using url query strings to display filtered data without requiring user input.

One other note.   FA does call some scripts (like supplier_inquiry.php from po entry to display the credit total) with an url query string.    When called this way, the script suppresses the FA headers.  But headers are shown in the ajax/browser sync.

So the lazy hack I made to discern the two different cases was to check if one of the newly added fields was missing.   This really should be solved instead by passing a flag to suppress the header.    To do this, one would have to identify all the calling scripts and set the suppress_header flag.